integreat_cms/cms/views/pois/poi_actions.py

```diff
@@ -42,7 +42,7 @@ def archive_poi(
     """
     poi = POI.objects.get(id=poi_id)
 
-    if poi.events.count() > 0:
+    if poi.events.count() > 0 or poi.contacts.count() > 0:
```

Comment:
I think it would be better to include that check in `poi.archive`, so that it works the same as `poi.delete` (Also we can use `poi.is_used` here)

Reply:
I think I will change this line to `is_used` then, I think this is the better solution